modules/installation-configuration-parameters.adoc

```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          referenceName:
+|Optional. The identifier of the storage resource configured in the failure domain. If you use this identifier, you must configure a matching failure domain storage resource. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          uuid:
+|The UUID of the storage resource in Prism Central. This value is required if you attach an additional storage resource to virtual compute machines.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          adapterType:
+|The adapter type of the storage resource. If the disk type is "Disk", valid values are "SCSI", "IDE", "PCI", "SATA" or "SPAPR".
+If the disk type is "CDRom", valid values are "IDE" or "SATA".
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          deviceIndex:
+|The index of the storage resource. The device index for disks that share the same adapter type should start at 0 and increase consecutively. The default value is 0. For each virtual machine, the `Disk.SCSI.0` and `CDRom.IDE.0` indices are reserved. If you use the `Disk.SCSI` or `CDRom.IDE` disk and adapter types, the `deviceIndex` should start at 1.
+|Non-negative integer, including 0.
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        deviceProperties:
+          deviceType:
+|The device type of the storage resource. Valid values are "Disk" and "CDRom". The default value is "Disk".
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        diskSize:
+|The size of the disk to attach to the virtual machine. The minimum size is 1Gb.
+|Quantity format, such as 100G or 100Gi. For more information on this format, see link:https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Format.
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        storageConfig:
+          diskMode:
+|The disk mode of the storage resource. Valid values are "Standard" or "Flash", and the default is "Standard".
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        storageConfig:
+          storageContainer:
+            name:
+|Optional. The name of the storage container object used by the virtual machine disk in Prism Central.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        storageConfig:
+          storageContainer:
+            referenceName:
+|Optional. The reference name of the storage container in the failure domain. If you use this identifier, you must configure a matching failure domain storage resource. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.
```

Comment:
Optional. The reference name of the storage container in the failure domain. If you use this, you must configure a matching storageContainer (with the same reference name) in each failure domain the compute nodes will spread out.  For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.